wdl/GatherBatchEvidence.wdl

```diff
@@ -214,6 +215,14 @@ workflow GatherBatchEvidence {
   }
 
   Array[String] samples_batch = select_first([ref_panel_samples, samples])
+  call util.ValidatePedFile {
+    input:
+      ped_file = ped_file,
+      sample_list = write_lines(samples_batch),
+      sv_pipeline_docker = sv_pipeline_docker,
+      runtime_attr_override = runtime_attr_validate_ped
+  }
```

Comment:
I think it would good to require this passes before attempting to run cnmops, otherwise you would get two errors with a bad ped file (1 from validation and 1 from cnmops, which could be confusing). If you let the validated ped file be the output, you can just wire that up to the ped file subset task.

Reply:
Good point!